Updated instructions on upload credentials. Fixes #393 (#395)
ebi-ait/hca-ebi-wrangler-central: docs/SOPs/contributor_communication_SOP.md

> 
> 2 Upload your data to our cloud space
> 

> > The credentials you will need will be sent separately by a developer in our team.

Reviewer: not necessarily a developer. Ideally, this would be automated.
```suggestion
> The credentials you will need will be sent separately by a member of the inges team.
```

Author: ```suggestion
> The credentials you will need will be sent in a separate email.
```
Actually, they don't need to know the role of the person that will send the email at all

> 
> On behalf of the HCA DCP Data Wrangling Team

### Contributor credentials and upload area

> Dear [name of contributor assigned in the ticket],
>
> Attached at the end of the email you will find a zip file with the credentials you need to upload your data.
> 

> > Your upload area ID is: [Generated upload area UUID]

Reviewer: I see that not all upload areas are named as UUID. 
We could use a more informative naming pattern like: 

- `contributor-id`-`short-name`  
- `contributor-id`-`short-name`-`uuid`
- `contributor-id`-`uuid`

Author: @amnonkhen they are not named as UUID but they all can be accessed using the UUID with the command `hca-util select <uuid>`

>
> The password to open the zip file is: [password]
>
> Once decompressed, you will find the credentials you need within the JSON file:
>
> AWS Access key: AccessKeyId field
>
> Aws Secret Access Key: SecretAccessKey field
>
> To upload your raw data files to our cloud space you will need to follow the instructions here: [How to upload data to an hca upload area with hca-util](https://github.com/ebi-ait/hca-documentation/wiki/How-to-upload-data-to-an-upload-area-using-hca-util) 
> 
> Many thanks for your contribution to the Human Cell Atlas
>
> [Developer name]
>

> > On behalf of the HCA DCP Data Wrangling Team

Reviewer: ```suggestion
> On behalf of the HCA DCP Data Wrangling Team
> View all HCA projects in the [HCA Project Catalogue](https://www.ebi.ac.uk/humancellatlas/project-catalogue)
```
